src/modules/fw_att_control/FixedwingAttitudeControl.cpp

```diff
@@ -574,6 +602,10 @@ void FixedwingAttitudeControl::run()
 					perf_count(_nonfinite_input_perf);
 				}
 
+				if (_vehicle_status.in_transition_mode || !_vehicle_status.is_rotary_wing) {
```

Comment:
@dagar This is kind of a gain scheduling. In transition and fw mode the fw controller gains get scaled down as if the vehicle was flying at max airspeed.

Reply:
Should be tailsitter only though right? I think we should get all the vtol (and tailsitter) special cases centralized in a function outside of the main loop. Then a single check for tailsitter that corrects everything (attitude, control mode, airspeed).